packages/sdk/src/lib/dataProtectorCore/processProtectedData.ts

```diff
@@ -33,10 +33,12 @@ import { IExecConsumer } from '../types/internalTypes.js';
 import { getWhitelistContract } from './smartContract/getWhitelistContract.js';
 import { isAddressInWhitelist } from './smartContract/whitelistContract.read.js';
 
+export type ProcessProtectedData = typeof processProtectedData;
+
 export const processProtectedData = async ({
   iexec = throwIfMissing(),
-  protectedData = throwIfMissing(),
-  app = throwIfMissing(),
+  protectedData,
+  app,
```

Comment:
Let `yup` give us a proper "required" error instead of a very generic "Missing parameter".